cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,18 @@
+describe("Login Form", () => {
+  it("Logs in the user with valid credentials", () => {
+    cy.visit("https://weronikamartinsen.github.io/social-media-client/")
```

Comment:
```suggestion
    cy.visit("/")
```

You must test the code in the current branch, not the code that has been deployed and merged already.